cf-agent/verify_files.c

```diff
@@ -624,20 +624,19 @@ static bool SaveBufferCallback(const char *dest_filename, void *param, NewLineMo
     return true;
 }
 
-
-static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp, const Attributes *attr,
+static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp, const Attributes *a,
```

Comment:
I would keep `attr` to be consistent with most of the other functions signatures

Reply:
@lluiscampos Thanks, I agree, fixed.